src/test/components/AppLocalizationProvider.test.js

```diff
@@ -134,4 +148,53 @@ describe('AppLocalizationProvider', () => {
     // $FlowExpectError Our version of flow doesn't know about document.dir
     expect(document.dir).toBe('rtl');
   });
+
+  it('fetches FTL strings for all available locales, but not others', async () => {
+    const { store, translatedText } = setup({
+      // At the time of this  writing, "de" is available but "fr-QC" isn't.
+      // Please replace the languages in this test if this changes in the future.
+      languages: ['de', 'fr-QC'],
```

Comment:
Hmm, this looks like a bit error prone in the future. Does it makes sense to invent our own locale? 😄 But not sure if that's possible with Fluent. So I'm okay if we land it like this as well.

Reply:
Yeah I wasn't sure... I asked in the Fluent channel, and if I don't get an answer today I'll still land this. I think this would be easy to change if necessary... (maybe when fr is enabled? not sure)

Reply:
I replaced with ab-CD like flod suggested.

Reply:
Actually then zibi suggested to use `und` for `undefined` :)